sn_networking/src/event.rs

```diff
@@ -487,10 +495,29 @@ impl SwarmDriver {
                     ..
                 } => {
                     trace!("Received request {request_id:?} from peer {peer:?}, req: {request:?}");
-                    self.send_event(NetworkEvent::RequestReceived {
-                        req: request,
-                        channel: MsgResponder::FromPeer(channel),
-                    })
+                    // if the request is replication, we can handle it and send the OK response here,
+                    // as we send that regardless of how we handle the request as its unimportant to the sender.
+                    match request {
+                        Request::Cmd(cmd) => {
```

Comment:
sounds like the `Replicate` is the only `Request::Cmd` to be received across nodes?
I'd prefer narrow down then enum to be `Cmd::Replicate` only, to avoid later on it mess-up with other new work flow ?